global_routes.go

```diff
@@ -0,0 +1,203 @@
+package main
+
+import (
+	"fmt"
+	"strings"
+
+	"github.com/Clever/amazon-kinesis-client-go/decode"
+)
+
+const defaultValueField = "value"
+const statTypeCounter = "counter"
+const statTypeGauge = "gauge"
+
+func globalRoutes(fields map[string]interface{}) []decode.AlertRoute {
+	routes := []decode.AlertRoute{}
+
+	// chain and append all global routes here
+	routes = append(routes, processMetricsRoutes(fields)...)
+	routes = append(routes, rsyslogRateLimitRoutes(fields)...)
+	routes = append(routes, gearmanRoutes(fields)...)
+	routes = append(routes, gearcmdPassfailRoutes(fields)...)
+	routes = append(routes, gearcmdHeartbeatRoutes(fields)...)
+	routes = append(routes, wagCircuitBreakerRoutes(fields)...)
```

Comment:
I like this organization :100:

Also don't forget the gearcmd.duration rule: https://github.com/Clever/log-router/blob/master/config/data.toml#L50-L59